CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -1097,6 +1097,38 @@ static function ValidateXpMultiplers ()
 	}
 }
 
+static function GrantBonusStartUpStaff (XComGameState StartState)
+{
+	local XComGameState_HeadquartersXCom XComHQ;
+	local XComGameState_Unit EngineerState, ScientistState;
+
+	XComHQ = XComGameState_HeadquartersXCom(`XCOMHISTORY.GetSingleGameStateObjectForClass(class'XComGameState_HeadquartersXCom'));
+	
+	if (XComHQ == none)
+	{
+		return;
+	}
+
+	EngineerState = `CHARACTERPOOLMGR.CreateCharacter(StartState, eCPSM_Mixed, 'Engineer');
+	ScientistState = `CHARACTERPOOLMGR.CreateCharacter(StartState, eCPSM_Mixed, 'Scientist');
+
+	EngineerState.SetSkillLevel(5);
+	ScientistState.SetSkillLevel(5);
```

Comment:
Is that the default skill level? We want them to be normal guys

Reply:
yes that is default i'll add a comment